frontend/src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { RouterProvider } from "react-router-dom";
+
+import Input from "./browser-router.js";
+
+function App() {
```

Comment:
Also please always use arrow functions instead of regular functions whenever possible.